app/views/tests/_question.html.erb

```diff
@@ -0,0 +1,62 @@
+<div class="col-md-4">
+  <div class="form-group">
+    <p><strong>2. Tôi thích gặp gỡ, làm việc với con người</strong></p>
+    <div>
+      <div class="form-check">
+        <input class="form-check-input" type="radio"
+          name="answer[4][29]"
+          id="answer4291"
+          value="1" oninvalid="this
+          .setCustomValidity('Bạn chưa trả lời')"
+          onvalid="this.setCustomValidity('')" required checked>
+        <label class="form-check-label" for="answer4291">
+        Không đúng (1đ)
+        </label>
+      </div>
+      <div class="form-check">
+        <input class="form-check-input" type="radio"
+          name="answer[4][29]"
+          id="answer4292"
+          value="2" oninvalid="this
+          .setCustomValidity('Bạn chưa trả lời')"
```

Comment:
không viết javascript lồng vào html thế này, tách toàn bộ javascript vào các file js riêng